### ISSUE TITLE
Bug in the Loss Function

### ISSUE THREAD

**Issue by saketguru** [theeluwin/pytorch-sgns#12]
The loss function currently implement is 
`-(oloss + nloss).mean()`

It should be
`(-oloss + nloss).mean()`

You want to minimize the distance between "positive samples" and maximize the distance between "negative samples".

**Comment by theeluwin** [comment]
As you would've noticed, there's `.neg()` chain at the end of `nvectors` definition (https://github.com/theeluwin/pytorch-sgns/blob/master/model.py#L72).
But this does seem confusing. I'll fix it to be more verbose someday.

**Comment by saketguru** [comment]
Yes. I noticed it. Thanks for the response.
